core/src/test/java/org/bitcoinj/wallet/DefaultCoinSelectorTest.java

```diff
@@ -54,7 +54,7 @@ public void selectable() throws Exception {
         t.getConfidence().setSource(TransactionConfidence.Source.SELF);
         assertFalse(DefaultCoinSelector.isSelectable(t));
         t.getConfidence().markBroadcastBy(new PeerAddress(UNITTEST, InetAddress.getByName("1.2.3.4")));
-        assertFalse(DefaultCoinSelector.isSelectable(t));
+        assertTrue(DefaultCoinSelector.isSelectable(t));
```

Comment:
Was it a bug in the test?

Reply:
The change in the test is because of #de65f061006a88e8556072f871f5c634e8054525, which changed the result of `isSelectable`